src/qt/locale/bitcoin_zh_TW.ts

```diff
@@ -2774,8 +2774,8 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis p
         <translation>簽名訊息時給出的簽名</translation>
```

Comment:
retouch the sentence.
```suggestion
        <translation>簽署訊息時給出的簽章</translation>
```